### PR TITLE
RCT/YJ/Fix 5-34 bug

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
I addressed issue #1586 in this PR. Thanks!

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section5/section5rule34.py [hunk 1]
```diff
@@ -52,20 +52,20 @@ def get_calc_vals(self, context, data=None):
...
+                        or p_zone_infiltration["modeling_method"]
+                        != b_zone_infiltration["modeling_method"]
+                    ):
+                        failing_infiltration_zone_ids.append(p_zone["id"])
 
             return {"failing_infiltration_zone_ids": failing_infiltration_zone_ids}
 
         def rule_check(self, context, calc_vals=None, data=None):
-            return len(calc_vals["failing_infiltration_zone_ids"]) == 0
+            return not calc_vals["failing_infiltration_zone_ids"]
```
Comment (yunjoonjung-PNNL) [inline rct229/rulesets/ashrae9012019/section5/section5rule34.py:71]: I simplified this part because empty list is falsey in Python.

**Review by weilixu — APPROVED** [review]

File: rct229/rulesets/ashrae9012019/section5/section5rule34.py [hunk 2]
```diff
@@ -52,20 +52,20 @@ def get_calc_vals(self, context, data=None):
             matched_baseline_zones = match_lists_by_id(proposed_zones, baseline_zones)
             proposed_baseline_zone_pairs = zip(proposed_zones, matched_baseline_zones)
             for p_zone, b_zone in proposed_baseline_zone_pairs:
-                # need a method like match object
-                p_zone_infiltration = p_zone["infiltration"]
-                # b_zone could be NONE - add a check.
-                b_zone_infiltration = b_zone["infiltration"]
+                if b_zone is not None:
```
Comment (JacksonJ-KC) [inline rct229/rulesets/ashrae9012019/section5/section5rule34.py:55]: what is happening if b_zone is `None`?

Comment (JacksonJ-KC) [reply rct229/rulesets/ashrae9012019/section5/section5rule34.py:55]: @weilixu @yunjoonjung-PNNL I think the case where a match cannot be made between baseline and proposed zones needs to be handled explicitly as an undetermined case.

Comment (yunjoonjung-PNNL) [reply rct229/rulesets/ashrae9012019/section5/section5rule34.py:55]: Addressed in PR #1593.

**Pull request merged**